CHANGES

```diff
@@ -3,6 +3,17 @@
 ===============================================================================
 # Complete support MPI 4.1 specification
 
+# Experimental threadd communicator feature (e.g. MPIX_Threadcomm_init).
+  See paper "Frustrated With MPI+Threads? Try MPIxThreads!",
+  https://doi.org/10.1145/3615318.3615320.
+
+# Experimental datatype functions MPIX_Type_iov_len and MPIX_Type_Iov
+
+# Experimental op MPIX_EQUAL for MPI_Reduce and MPI_Allreduce (intra
+  communicator only)
+
+# CH4 shm is enabled by default with ucx netmod.
+
```

Comment:
```suggestion
```

Reply:
Leaked back in during rebase :( Good catch.